Gemfile

```diff
@@ -1,9 +1,11 @@
 source "https://rubygems.org"
-ruby RUBY_VERSION
+ruby file: ".ruby-version"
```

Comment:
Instead of getting the ruby version from the local install, explicitly specify a version that matches the one used in the `build-site` GitHub action (to ensure we dev on same version of Ruby used by the automated daily build).